opencl-aot/CMakeLists.txt

```diff
@@ -22,6 +26,7 @@ if (NOT OpenCL_INCLUDE_DIRS)
             COMMENT "Downloading OpenCL headers."
             )
 else ()
+    message("Copying OpenCL headers...")
```

Comment:
What is this for?

Reply:
This is for notification about copying headers/ICD on configure step. Should it be removed?

Reply:
I think it's confusing. If I understand correctly headers are not copied during build files configuration. It looks like some debug messages... 
I think it's better to remove or rephrase.

Reply:
Created PR https://github.com/intel/llvm/pull/959 with fix